client/components/plans/plan/index.jsx

```diff
@@ -249,6 +250,25 @@ module.exports = React.createClass( {
 		);
 	},
 
+	getFreeDomainMessage: function() {
+		if ( abtest( 'promoteFreeDomain' ) !== 'freeDomain' ) {
+			return null;
+		}
+
+		if ( this.props.plan && isPlan( this.props.plan ) ) {
+			return (
+				<div className="plan__free-domain-message">
+					<Gridicon icon="checkmark" size="12" />
+					{ this.translate( 'Includes a free domain' ) }
+				</div>
+			);
+		}
+
+		return (
+			<div className="plan__free-plan-message">&nbsp;</div>
```

Comment:
Since this a test that will only live a week or so I guess it's fine to have this duplicate code.